### DISCUSSION TITLE
Proxying to non-dockerised service

### DISCUSSION THREAD

**Discussion by sjtrny** [nginx-proxy/nginx-proxy#2370] [General]
Hi, I currently have a server running webmin on port 10000 outside of Docker and other services running in Docker containers.

Is it possible to have nginx-proxy forward all requests that don't match `VIRTUAL_HOST` to this webmin service? I've tried using `DEFAULT_ROOT` but nginx-proxy always returns 503 which indicates to me that it thinks a `VIRTUAL_HOST` matches but this mystical upstream service doesn't exist.

Here's a diagram of my setup:

- `nas.local` (server)
  - `10000` -> webmin standalone
  - `80` -> nginx-proxy container
    - `portainer.nas.local` -> portainer container
    - `sftpgo.nas.local` -> sftpgo container
    - ...

What I want is for requests to `nas.local:80` be redirected to `nas.local:10000`.

**Comment by sokai** [comment]
Thanks for this discussion, @sjtrny !

I only wanna add my use case (to the origin example), which implies IMHO the same question.

I'd like to be able to redirect (HTTP + HTTPS) request in the nginx-proxy container for a third sub-domain (in the origin example) `third.nas.local:80` to some other ports on the server/host/machine, like this:

 * nas.local` (server)
  * `10000` → webmin standalone
  * `80` + `443` → nginx-proxy container
    * `portainer.nas.local` → portainer container
    * `sftpgo.nas.local` → sftpgo container
    * `third.nas.local` → `third.nas.local:1080`/`localhost:1080`
    * …
  * `third.nas.local:1080`/`localhost:1080` → service (on server)

Maybe some of the *main* devs have any clue: @SchoNie or @buchdag ?

Thanks a lot in advance and KR!
